Source/Core/Core/HW/DSPHLE/DSPHLE.cpp

```diff
@@ -79,15 +79,16 @@ void DSPHLE::SwapUCode(u32 crc)
 {
```

Comment:
as the comment above states, this was always kinda hacky. Now the behavior doesn't really match the comment, either. Maybe a list should be kept (instead of implicit 2-entry list)?